backend/supabase/groups/indexes.sql

```diff
@@ -0,0 +1,5 @@
+
+create index if not exists group_slug on public.groups (slug);
+create index if not exists group_name on public.groups (name);
```

Comment:
Nice, looks like this index is working:
<img width="1076" alt="Screenshot 2023-05-16 at 9 21 10 AM" src="https://github.com/manifoldmarkets/manifold/assets/23196210/db189c23-e93f-4629-a008-24921264eb94">